component/src/main/java/io/siddhi/extension/io/file/FileAlterationImpl.java

```diff
@@ -0,0 +1,123 @@
+/*
+ * Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
```

Comment:
Shall we update the license headers?

Reply:
fixed in https://github.com/siddhi-io/siddhi-io-file/tree/5f2348a3cec4571d14c1f3f939dfddcbadd466e8